src/libcrun/linux.c

```diff
@@ -4082,7 +4083,13 @@ maybe_get_idmapped_mount (runtime_spec_schema_config_schema *def, runtime_spec_s
 
   ret = syscall_mount_setattr (newfs_fd, "", AT_EMPTY_PATH | (recursive ? AT_RECURSIVE : 0), &attr);
   if (UNLIKELY (ret < 0))
-    return crun_make_error (err, errno, "mount_setattr `%s`", mnt->destination);
+    {
+      if (errno == EINVAL)
+        {
+          extraMsg = "(maybe the file system used doesn't support idmap mounts on this kernel?)";
```

Comment:
should we prepend the empty space before the `(`, and use `"mount_setattr `%s`%s"` below so we don't end up with an extra space if extraMsg is empty?

nit: can you please rename the variable to `extra_msg`?

Reply:
> should we prepend the empty space before the `(`, and use `"mount_setattr `%s`%s"` below so we don't end up with an extra space if extraMsg is empty?

I doubted a lot about doing that. Great that you have a preference, I was unsure. I'll update the PR.



> nit: can you please rename the variable to `extra_msg`?

Yes, sorry. I've just wrote the go patch and didn't realize about the style :)